### PR TITLE
Fix: Kotlin --> Wasm now has parity with Cpp --> Wasm

### PR DESCRIPTION
**Pull request by alxrsngrtn**
## Summary
- Added README for building Kotlin Wasm particles
- Followed @mykmartin 's pattern to allow for multi-particle shared env
- changed `requestRender` to `renderSlot`. 

## How to Verify
- Follow the README instructions
- Expected: Should see the Kotlin / Wasm harness that @cromwellian built.

### CODE REVIEW

File: src/wasm/kotlin/wasm/arcs/Particle.kt [hunk 1]
```diff
@@ -48,22 +49,37 @@ abstract class Particle : WasmObject() {
...
+      renderSlot(slotName)
     }
+
+    open fun getTemplate(slotName: String): String = ""
+    open fun populateModel(slotName: String, accumulateModel: (String, String) -> Unit): String = ""
 }
 
 abstract class Handle : WasmObject() {
     var name: String? = null
+    var particle_: Particle? = null
```
Comment (shaper) [inline src/wasm/kotlin/wasm/arcs/Particle.kt]: Is the '_' suffix a standard (/ our standard)? for internal data?

File: src/wasm/kotlin/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Kotlin --> WASM particles
+
+## Build & Test
+1. [`npm install && ./tools/sign webpack`](../../../README.md#install)
```
[the remark this answers is not in the document]
Comment (alxrsngrtn) [reply src/wasm/kotlin/README.md]: Thank you. Good catch.

File: src/wasm/kotlin/wasm/arcs/Particle.kt (hunk 1 above)
Comment (alxrsngrtn) [reply src/wasm/kotlin/wasm/arcs/Particle.kt]: I was following a convention from cpp. There's no precedent in the file for that, so I'll drop the `_`.

**Comment by cromwellian** [comment]
Great work Alex, LGTM

**Pull request merged**